src/sentry/integrations/jira/webhooks/uninstalled.py

```diff
@@ -15,15 +16,11 @@
 
 @control_silo_endpoint
 class JiraSentryUninstalledWebhook(JiraWebhookBase):
-    owner = ApiOwner.INTEGRATIONS
-    publish_status = {
-        "POST": ApiPublishStatus.PRIVATE,
-    }
     """
     Webhook hit by Jira whenever someone uninstalls the Sentry integration from their Jira instance.
     """
 
-    def post(self, request: Request, *args, **kwargs) -> Response:
+    def authenticate(self, request: Request, **kwargs) -> Any:
```

Comment:
ideally authenticating the incoming request would be on its own, but the helper function decodes the token, fetches the integration, and verifies the token (possibly using information from `integration.metadata`), so it's not easy to decouple